environments/dev-env/CONTRIBUTING.md

```diff
@@ -0,0 +1,510 @@
+# KCP-Edge Contributor Development Environment
+
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl  
+
+
+## Supported OS Platforms 
+  - Linux
+  - MacOS
+  - Windows WSL/Ubuntu
+
+
+Run the following script to install the required package (Linux or MacOS ):
+
+```bash
+./install_req.sh
+```
+
+Alternatively, run the following command for MacOS platform:
+
+```bash
+brew install ko gcc jq make go@1.19 kind kubectl
+```
+
+For Windows WSL/Ubuntu platform, follow the instructions [here](docs/wls_ubuntu_platform.md)
+
+
+## Quickstart
+
+### 1. Clone the kcp-edge repo:
+
+```shell
+git clone https://github.com/kcp-dev/edge-mc.git  kcp-edge
```

Comment:
(1) This has to come before the local usage of `install_req.sh` above.

(2) Must we mandate the renaming of the directory?  I think not.  This just adds extra dissonance.  Developers know that the git repo is named "edge-mc".  You might wish that the repo were named something different, and in the future it might even have a kcp-free name that we have not settled on yet, but at present the least dissonant thing to do is `git clone` without renaming.

Reply:
(1) added a comment stating that the script needs to be run after cloning the repo. I am avoiding to add a step to run the `install_req.sh` script in our instructions -  most of the developers will already have those packages installed in their environment or they will know how to install themselves. Our `install_req.sh` is just an option that we are providing to install the required packages

(2) Fixed